turbo.md

```diff
@@ -109,6 +137,29 @@ _start() => error: host function trapped
 
 Note that the error status reported is expected!
 
+* * *
+
+## WABT Overview
+
+The [WebAssembly Binary Toolkit](https://github.com/WebAssembly/wabt) (WABT) is a suite
+of tools for working with WebAssembly.  The tools are written in C/C++ and are designed
+for easy integration into other projects.
+
+The tool we will be focused on for this tutorial is **wasm-interp**.  This is a
```

Comment:
I wonder if it's easier to just say that "libc-interp" is the tool being worked on, to avoid confusion between `wasm-interp` and `libc-interp`.

Reply:
Agreed.  Re-pushed a new wording.